src/cpl.rs

```diff
@@ -125,7 +228,121 @@ impl Clone for CslStringList {
     }
 }
 
+impl Debug for CslStringList {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        let mut b = f.debug_tuple("CslStringList");
+
+        for e in self.iter() {
+            b.field(&e.to_string());
+        }
+
+        b.finish()
+    }
+}
+
+impl Display for CslStringList {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        // CSLPrint would be preferable here, but it can only write to a file descriptor.
+        let len = self.len();
+        for i in 0..len {
+            if let Some(field) = self.get_field(i) {
+                f.write_fmt(format_args!("{field}\n"))?;
+            }
+        }
+        Ok(())
+    }
+}
+
+impl PartialEq for CslStringList {
```

Comment:
Maybe we shouldn't implement this? Users might have different expectations here, like lists with a different order of elements being equal.

Reply:
I did it for the tests and doc-tests. So, for example, without this we'd have to change `assert_eq!(sl1, sl2);` to `assert_eq!(sl1.to_string(), sl2.to_string());`.  Think that's ok?

Reply:
But users will use it!